plugin/adsk/scripts/AETemplateHelpers.py

```diff
@@ -109,6 +110,16 @@ def GetStageFromProxyShapeAttr(attr):
 
     return(stageName, proxyStage)
 
+def GetFullStageNameFromProxyShapeAttr(attr):
+    # Helper method which returns the full stage name
+    # First get the stage name from the input attribute.
+    stageName = attr.split('.')[0]
+    # Convert that into a long Maya path so we can get the USD stage.
+    res = cmds.ls(stageName, l=True)
+    fullStageName = res[0]
+    
+    return(fullStageName)
```

Comment:
Why was this function needed? Couldn't you just have called the one above and ignore the stage?
`fullStageName,_ = GetStageFromProxyShapeAttr(filePathAttr)`

Reply:
Fair point. I have an upcoming PR and I am taking note to address this.

Reply:
Okay thanks. I'll merge this one then.

Reply:
Thank you